ftl/common/builder.py

```diff
@@ -123,8 +123,16 @@ def __init__(self, ctx, namespace, args, cache_version_str,
         self._target_creds = docker_creds.DefaultKeychain.Resolve(
             self._target_image)
         self._transport = transport_pool.Http(httplib2.Http, size=_THREADS)
-        self._base_image = docker_image.FromRegistry(
-            self._base_name, self._base_creds, self._transport)
+        if args.tar_base_image_path:
+            try:
+                self._base_image = docker_image.FromTarball(
+                    args.tar_base_image_path)
+            except Exception as e:
+                logging.error(e)
```

Comment:
Maybe logging.fatal?

I would actually just drop this try/except since the constructor doesn't actually do anything:
https://github.com/google/containerregistry/blob/master/client/v2_2/docker_image_.py#L217